deploy/crds/toolchain_v1alpha1_usersignup.yaml

```diff
@@ -11,6 +11,10 @@ spec:
     name: User ID
     priority: 1
     type: string
+  - JSONPath: .spec.username
+    name: Username
+    priority: 1
+    type: string
```

Comment:
this should be generated from `api` repo - see my comment there: https://github.com/codeready-toolchain/api/pull/31#pullrequestreview-282615206